packages/rocketchat-ui-login/login/form.coffee

```diff
@@ -112,15 +112,16 @@ Template.loginForm.events
 
 				Meteor[loginMethod] formData.emailOrUsername, formData.pass, (error) ->
 					RocketChat.Button.reset(button)
-					if error?
-						if error.error is 'no-valid-email'
-							instance.state.set 'email-verification'
-						else
-							toastr.error t 'User_not_found_or_incorrect_password'
-						return
+					loginMethod = 'loginWithPassword'
+					Meteor[loginMethod] formData.emailOrUsername, formData.pass, (error) ->
```

Comment:
this code is always calling *both* login methods, even if the first one (LDAP) succed..
I think the fallback call should be done only if the first returns an error..